frontend/src/components/complete-set-table/complete-set-table.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+
+import { CarSetOptions } from '@common/enums/enums';
+import { CompleteSetPropsType } from '@common/types/types';
+import { OptionsIcon } from '@components/common/icons/icons';
+import {
+  TableContainer,
+  Paper,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+} from '@mui/material';
+import { clsx } from 'clsx';
+
+import { mockCars } from './mock-cars';
+import styles from './styles.module.scss';
+
+const CompleteSetTable: React.FC = () => {
+  const [cars] = useState<CompleteSetPropsType[]>(mockCars);
+
+  return (
+    <TableContainer
+      component={Paper}
+      elevation={0}
+      className={clsx(styles.tableContainer, styles.container)}
+    >
+      <Table
+        sx={{ minWidth: 650 }}
+        aria-label="complete set table"
+        className={styles.table}
+      >
+        <TableHead>
+          <TableRow>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.Model}
+            </TableCell>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.Complectation}
+            </TableCell>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.Color}
+            </TableCell>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.Motor}
+            </TableCell>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.WheelDrive}
+            </TableCell>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.Race}
+            </TableCell>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.Price}
+            </TableCell>
+            <TableCell className={styles.tableTitle}>
+              {CarSetOptions.Compare}
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {cars.map((car) => (
+            <TableRow key={car.id} className={styles.tableRow}>
+              <TableCell className={styles.tableRow}>
+                {car.brand} {car.model}
+              </TableCell>
+              <TableCell className={styles.tableRow}>
+                {car.complectation}
+              </TableCell>
+              <TableCell className={styles.tableRow}>
+                <div
+                  className={styles.colorBox}
+                  style={{ backgroundColor: car.color }}
+                ></div>
```

Comment:
`div` can be collapsed: 
```suggestion
                <div
                  className={styles.colorBox}
                  style={{ backgroundColor: car.color }}
                />
```